config/v1/types_build.go

```diff
@@ -32,54 +32,65 @@ type BuildSpec struct {
 }
 
 type BuildDefaults struct {
+	// WebProxy is the proxy settings for web traffic (HTTP/HTTPS)
+	// +optional
+	WebProxy Proxy `json:"webProxy,omitempty"`
```

Comment:
let's call it RegistryProxy

people may want different proxies for different things and arguably git is also web traffic in some cases, so calling it webproxy is confusing